test/NameRegistry.t.sol

```diff
@@ -1625,6 +1678,45 @@ contract NameRegistryTest is Test {
         assertEq(nameRegistry.recoveryOf(ALICE_TOKEN_ID), recovery);
     }
 
+    function testFuzzCannotSafeTransferFromApproverIfPaused(
+        address alice,
+        address bob,
+        address approver,
+        address recovery
+    ) public {
+        _assumeClean(alice);
+        _assumeClean(recovery);
+        _assumeClean(approver);
+        vm.assume(bob != address(0));
+        vm.assume(alice != bob);
```

Comment:
nit: do all the alice assertions in one line like `vm.assume(alice != bob && alice != approver);`

i know some of the existing tests also have this issue, ive been meaning to clean them up